src/config/index.ts

```diff
@@ -59,6 +60,7 @@ export const SOLD_LAND_FILTER_PAGE = "sold-land-filter";
 export const TRUST_INFO_PAGE = "trust-information";
 export const USE_PAPER_PAGE = "use-paper";
 export const WHO_IS_MAKING_FILING_PAGE = "who-is-making-filing";
+export const SIGN_OUT_PAGE = "sign-out";
```

Comment:
I think this represents the page where the user confirms they wish to sign out? If yes, suggest a minor name change here (and possibly to other code artefacts below) to make this clear. Could name the constant CONFIRM_SIGN_OUT_PAGE.

Reply:
Same name in the url and file name. The same as for the other template name, so I will keep it consistent.

Reply:
Thanks Martin